### DISCUSSION TITLE
How to disable the user location blue dot?

### DISCUSSION THREAD

**Discussion by johanneslinnama** [react-native-maps/react-native-maps#4856] [Q&A]
Hey,

I'm using showsUserLocation={true} with onUserLocationChange to follow the user on Android.

The functionality is great, but I'd like to disable the blue dot (and replace it with my custom one).

I'm aware there doesn't seem to be a built-in feature for this on Android, but where can I find the blue dot on source code so I can disable it from there?

Thanks!

**Comment by rishabh-synclovis** [comment]
I will also getting same issue. Let me know if you got any solution.

**Reply by johanneslinnama** [reply]
Didn't find a built in way to do it, but I did a workaround by having two maps. One is invisible with showsUserLocation enabled and provides location to the other map to move my custom location marker there.
